### ISSUE TITLE
Keyboard API

### ISSUE THREAD

**Issue by tafelito** [FaridSafi/react-native-gifted-chat#326]
Are you planning to remove the dependency for the dismissKeyboard library now that you can do the same with the React Native Keyboard API?

And actually, what's the benefit of dismissing the keyboard manually on the touchEnd event instead of just removing `keyboardShouldPersistTaps={true}` from the ListView?

**Comment by kfiroo** [comment]
@tafelito Yes, just need time to do it. PRs are welcome

Not sure why is that, maybe an old bug or something, if the behaviour is the same we should change it

**Comment by tafelito** [comment]
@kfiroo OK a PR is on the way!

**Comment by tafelito** [comment]
Ok PR ready. So I removed the dependency for `react-native-dismiss-keyboard` and removed the touch events. I'm still not quite sure why the Keyboard events are called, even if I don't add the listeners. But what I did see is that if I remove the Keyboard functions from the `invertibleScrollViewProps` then they are executed. I check the `react-native-invertible-scroll-view` plugin and there is no reference to the Keyboard, so I'm not sure why this is happening. Anyway, it works without the touch events
